client/layout/masterbar/style.scss

```diff
@@ -288,26 +288,40 @@ body.is-mobile-app-view {
 	justify-content: center;
 
 	&.masterbar__reader {
-		padding: 0 7px 0 4px;
+		padding: 0 10px 0 6px;
 
 		.masterbar_svg-reader {
-			padding: 0 8px 0 3px;
+			padding: 0 0 0 4px;
```

Comment:
@davemart-in it looks like your other PR is changing these items to different values than what is happening here.

There will be a merge conflict, but probably want to coordinate what we actually want for these paddings.

Reply:
Thanks for highlighting this @Addison-Stavlo!

@allilevine why don' you merge your changes, then I'll update my PR, I'll fix any merge conflicts, retest, and then ship my PR.

Reply:
Sounds good, thanks!